src/write/elf.rs

```diff
@@ -83,6 +83,7 @@ impl Object {
             Architecture::PowerPc64 => false,
             Architecture::Riscv64 => false,
             Architecture::Riscv32 => false,
+            Architecture::Sparc64 => false,
```

Comment:
Based on LLVM, this should be `true`, and same for some of the other architectures. I'll do a new PR to fix these up.